openhtf/io/output/mfg_inspector.py

```diff
@@ -252,10 +252,10 @@ def _ExtractParameters(record, testrun, used_parameter_names):
         testrun_code = testrun.failure_codes.add()
         testrun_code.code = testrun_param.name
         if measurement.dimensions is None:
-          if testrun_param.numeric_value:
-            testrun_code.details = str(testrun_param.numeric_value)
-          else:
+          if testrun_param.text_value:
```

Comment:
this is now going to be falsey if text_value is an empty string, which will change the behavior of this to make details the string "None" when the text value is an empty string, which is kind of weird and probably not what we want.  It's probably better to change the `if testrun_param.numeric_value:` to `if testrun_param.numeric_value is not None:`.